### PR TITLE
Base style

### PR DESCRIPTION
**Pull request by Quentin-BLANC**
Presentation, partners, legalmention.

### CODE REVIEW

**Comment by emrecaydin** [comment]
request done

File: public/assets/css/baseStyle.css [hunk 1]
```diff
@@ -79,7 +79,10 @@ html[data-theme='dark'] {
 
 body {
     background-color: var(--main-bg-color);
-    margin: 0;
+}
+
+h1 {
+    text-align: center;
 }
```
Comment (emrecaydin) [inline public/assets/css/baseStyle.css:87]: done

File: public/assets/css/partnersStyle.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+body {
+    text-align: center;
+}
```
Comment (emrecaydin) [inline public/assets/css/partnersStyle.css:3]: done

File: public/assets/css/presentationStyle.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    }
+    .desktop-version{
+        width: 100%;
+        text-align: center;
+    }
+    .desktop-version img {
+        width: 80%;
+    }
+}
+
```
Comment (emrecaydin) [inline public/assets/css/presentationStyle.css:43]: well done

**Review by emrecaydin — APPROVED** [review]
good job

**Review by RomainClair — APPROVED** [review]

**Pull request merged**